recipes/lzfse/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "lzfse" %}
+{% set version = "2017.03.08" %}
+{% set sha256hash = "68193df40e8bc8eb87ee7ec47f2da4be64d1c172d1ce558d38aefd0bdcc0ed51" %}
+{% set ghcommit = "88e2d2788b4021d0b2eb9fe2d97352ae9190f128" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ ghcommit }}.zip
```

Comment:
Could we please ask them to tag a release? Just something we can use reliably. Feel free to cc me if you wish.